.github/workflows/markdown-link-check.json

```diff
@@ -2,6 +2,10 @@
     "ignorePatterns": [
       {
         "pattern": "^(?!http(s)?:\/\/.*)|^(http(s)?:\/\/.*((osism.xyz)|(in-a-box.cloud)))(:?[0-9]+)?(\/.*)?$"
+      },
+      {
+        "pattern": "^https:\/\/www.wireguard.com$"
```

Comment:
This is not part of this PR. Do a proper rebase.

Reply:
Removed, just added for getting rid of the recurring problems with that domain.
After that i created the PR for the new link checker.